### ISSUE TITLE
Handling of blank lines is idiosyncratic

### ISSUE THREAD

**Issue by SteveGilham** [fsprojects/fantomas#311]
Using `fantomas . --recurse --indent 2 --pageWidth 90`, it usually elides empty lines which are in the code to separate stanzas (e.g. the Act/Arrange/Assert parts of a test, where I am happy for line rearrangement within the stanza), but will sometimes insert them where they did not exist before, and where they have no good reason to exist, such as

Before
```
                            [ Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/AltCover.Tests.dll"
                              Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/Sample2.dll" ])
```

After
```
                            [ Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/AltCover.Tests.dll"

                              Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/Sample2.dll" ])
```

**Comment by Ciantic** [comment]
I think this can be closed, I tried with [3.0 Preview tester](https://jindraivanek.gitlab.io/fantomas-ui/#?fantomas=preview-trivia&code=NoAgCghgLgFgdAcwKZQGIFcA2mByEC2SAsAFAjkWVXU2UBEA+gEICWAdhAE4tIDOA9AEFMUAMIB7AG5JOcACp8oAgCJIARugQBqQWwCeosAFV+DBkbYsoC3lADqVmMLFSZAJXRs2MoSInTZGyU4ABNsOlJaKOiKSFhEFAxsPEJImPTaRlYObj5fFwD5RRV1TR19QxMzCysgh1hnf3dPb05+AGUCAAdMJAAmUPCQAF0gA)

And it produced this:

```
[ Path.getFullName "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/AltCover.Tests.dll"
      Path.getFullName "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/Sample2.dll" ]
```

There is no additional new line there anymore in these versions.

**Comment by jindraivanek** [comment]
Closing, thanks @Ciantic.
